src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -0,0 +1,45 @@
+package com.crud.tasks.scheduler;
+
+import com.crud.tasks.config.AdminConfig;
+import com.crud.tasks.domain.Mail;
+import com.crud.tasks.repository.TaskRepository;
+import com.crud.tasks.service.SimpleEmailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailScheduler {
+    private static final String SUBJECT = "Tasks: Once a day email";
+
+    @Autowired
+    private SimpleEmailService simpleEmailService;
+
+    @Autowired
+    private TaskRepository taskRepository;
+
+    @Autowired
+    private AdminConfig adminConfig;
+
+    public String returnNumberOfTasks() {
+        long number = taskRepository.count();
+        if (number > 1) {
+            return "tasks";
+        } else if (number < 1) {
+            return "tasks";
+        } else {
+            return "task";
+        }
```

Comment:
fajnie ze wyciagnales to do zewnetrzej metody, ale:
- to powinna byc metoda prywatna + pamietaj metody prywatne pod publicznymi
- mozna to zapisac ktorej uzywajac tylko if-else (jesli bys sobie nie poradzil, daj prosze znac)